libs/utils/android/workloads/uibench.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, test_env):
         # Set of output data reported by UiBench
         self.db_file = None
 
-    def run(self, out_dir, test_name, duration_s, collect=''):
+    def run(self, out_dir, test_name, duration_s, collect='', action=''):
```

Comment:
Since tests are meant to be reproducible and shareable among users, would not be better to use a pre-defined a set of sensible actions for each test?

In that case it would probably be better to encode these reasonable set of actions in this test itself.

For example, we can use a dictionary `actions[TEST_NAME] = [action1, action2, ...]` to map the actions required by `test_TEST_NAME`.